var/spack/repos/builtin/packages/charliecloud/package.py

```diff
@@ -74,6 +75,7 @@ class Charliecloud(AutotoolsPackage):
         sha256="15ce63353afe1fc6bcc10979496a54fcd5628f997cb13c827c9fc7afb795bdc5",
     )
     variant("docs", default=False, description="Build man pages and html docs")
+    variant("squashfuse", default=False, description="Build with squashfuse support")
```

Comment:
Doe this work for old versions of `charliecloud`?

Reply:
Versions <=0.29 are marked as deprecated, so i did not test against those versions. It works for 0.31 and 0.30

As i am not the maintainer and as the charlycloud team talks about keeping it to minimum requirements i did not want to make it active by default